Cargo.toml

```diff
@@ -27,22 +31,23 @@ serde_json = "1.0"
 url = "2.1"
 codec = { package = "parity-scale-codec", version = "1.3", default-features = false, features = ["derive", "full"] }
 
-frame-metadata = { version = "11.0.0-rc2", package = "frame-metadata" }
-frame-support = { version = "2.0.0-rc2", package = "frame-support" }
-sp-runtime = { version = "2.0.0-rc2", package = "sp-runtime" }
-sp-version = { version = "2.0.0-rc2", package = "sp-version" }
-pallet-indices = { version = "2.0.0-rc2", package = "pallet-indices" }
+frame-metadata = { git = "https://github.com/paritytech/substrate", branch = "master" }
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-version = { git = "https://github.com/paritytech/substrate", branch = "master" }
+pallet-indices = { git = "https://github.com/paritytech/substrate", branch = "master" }
 hex = "0.4.0"
-sp-rpc = { version = "2.0.0-rc2", package = "sp-rpc" }
-sp-core = { version = "2.0.0-rc2", package = "sp-core" }
-sc-rpc-api = { version = "0.8.0-rc2", package = "sc-rpc-api" }
-sp-transaction-pool = { version = "2.0.0-rc2", package = "sp-transaction-pool" }
+sp-rpc = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-rpc-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch = "master" }
 substrate-subxt-proc-macro = { version = "0.8.0", path = "proc-macro" }
-sp-std = "2.0.0-rc2"
-sp-finality-grandpa = "2.0.0-rc2"
-sp-consensus-babe = "0.8.0-rc2"
-pallet-im-online = "2.0.0-rc2"
-sp-authority-discovery = "2.0.0-rc2"
+sp-std = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-finality-grandpa = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-consensus-babe = { git = "https://github.com/paritytech/substrate", branch = "master" }
+pallet-im-online = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-authority-discovery = { git = "https://github.com/paritytech/substrate", branch = "master" }
+polkadot = { git = "https://github.com/paritytech/polkadot", branch = "master", optional = true, default-features = false, features = ["std"], package = "polkadot-primitives" }
```

Comment:
We publish to crates.io so we can't have github dependencies, need to find out whether there are plans to publish `polkadot-primitives`